src/navinator.js

```diff
@@ -25,26 +26,35 @@
             var _ = this;
             _.defaults = {
                 buttonElement: '#navinator-button',
+                debug: false,
                 direction: 'left',
                 minWidth: 0,
-                navZ: 'bottom',
-                pageElement: '#page-container'
+                pageElement: '#page-container',
+                subNavElement: false
             };
 
             _.options = $.extend({}, _.defaults, settings);
 
             // Navinator Globals
+            _.el = element;
             _.$navElement = $(element);
             _.$pageElement = $(_.options.pageElement);
             _.$buttonElement = $(_.options.buttonElement);
             _.navCreated = false;
+            if (_.options.subNavElement) {
+                var subNavClassOrID = _.options.subNavElement.charAt(0);
+                var subNavEl = _.options.subNavElement.substring(1);
+                _.subNav = subNavClassOrID + 'navinator-' + subNavEl;
+            } else {
+                _.subNav = _.options.subNavElement;
```

Comment:
Can you default `_.subNav = false`? If you can, then you'd be able to get rid of this else statement.